cluster/examples/kubernetes/ceph/operator-openshift.yaml

```diff
@@ -88,6 +88,40 @@ users:
   - system:serviceaccount:rook-ceph:rook-csi-cephfs-plugin-sa
   - system:serviceaccount:rook-ceph:rook-csi-cephfs-provisioner-sa
 ---
+# Rook Ceph Operator Config
+# Use this ConfigMap to override operator configurations
+# Precedence will be given to this config in case
+# Env Var also exists for the same
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rook-ceph-operator-config
+  # should be in the namespace of the operator
+  namespace: rook-ceph
+data:
+  # (Optional) Ceph Provisioner NodeAffinity.
+  CSI_PROVISIONER_NODE_AFFINITY: "role=storage-node; storage=rook, ceph"
```

Comment:
Whatever is enabled in this configmap by default will be the new default settings. Shall we keep these defaults the same as what we had in the env vars in operator.yaml? In other words, seems like we should comment these settings out.

Reply:
Agreed. Commenting it out.